xamarin/Xamarin.Forms
# [Android] fix accessibility of Picker

### Description of Change ###

Changed accessibility of Picker. Now it reads like a `button`. If something is selected in it, this item is also read. This is part of PR https://github.com/xamarin/Xamarin.Forms/pull/3955

### Issues Resolved ### 

- fixes for Android #3454 

### API Changes ###
 
 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

Picker supports accessibility parameters without changing hint/placeholder of control.

### Before/After Screenshots ### 

Before
![image](https://user-images.githubusercontent.com/27482193/52205361-dde55100-2887-11e9-90db-bf0794b59cf4.png)

After
![image](https://user-images.githubusercontent.com/27482193/52205095-10427e80-2887-11e9-9b6e-f8e072375b3b.png)


### Testing Procedure ###

- run UItest 3454
- check accessibility of Pickers

### PR Checklist ###
